Snowflake.Data/Client/SnowflakeDbConnection.cs

```diff
@@ -22,6 +22,14 @@ public class SnowflakeDbConnection : DbConnection
 
         internal ConnectionState _connectionState;
 
+        private DbProviderFactory _snowflakeDbFactory = new SnowflakeDbFactory();

```

Comment:
why don't we call private field `_ dbProviderFactory` but with the name of specific class?

Reply:
right, but actually I'm removing since DbProviderFactory is already in the base class